.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deno.enable": true,
+  "deno.lint": true,
+  "deno.unstable": false
```

Comment:
We're using unstable features in the connector to support the NPM modules so should we try to match this with the dev tools?

Reply:
I'm not sure that refers to that, I think this enables access to Deno's unstable APIs. I've used an NPM package (for testing) in this project with that set to false and it works just fine.